backend/api/src/main/kotlin/de/keepmealive3d/event/LiveDataEventHandler.kt

```diff
@@ -1,7 +1,43 @@
 package de.keepmealive3d.event
 
-class LiveDataEventHandler: EventHandler {
+import com.influxdb.client.domain.WritePrecision
+import com.influxdb.client.kotlin.InfluxDBClientKotlinFactory
+import com.influxdb.client.write.Point
+import de.keepmealive3d.config.Config
+import kotlinx.coroutines.runBlocking
+import java.time.Instant
+
+class LiveDataEventHandler(conf: Config) : EventHandler {
```

Comment:
I would probably inject the config
```suggestion
class LiveDataEventHandler() : EventHandler, KoinComponent {
    val config: Config by inject()
```

Reply:
Done